### DISCUSSION TITLE
⚽⚽⚽please tell me your favorite football team! ⚽⚽⚽

### DISCUSSION THREAD

**Discussion by XC1996114514** [NAIST-SE/AttractingContributors#405] [General]
I need help with my football team list.⚽⚽⚽

To help you, please look at this list
The national football team: https://en.wikipedia.org/wiki/List_of_men%27s_national_association_football_teams
heck out the list of all the men and women football clubs broken down by country on fbref.com:  https://fbref.com/en/squads/

Come to my fork:
https://github.com/XianChen1996/AttractingContributors_type_in_your_favourite_footballteam

Add to my README file.

I look forward to your contributions

ps： If possible we can set up a NAIST football team and WATCH the world cup of 2022 🍻🍻

**Comment by AtropsCooper** [comment]
My favorite team added.
Help me please if you'd like to 🥺
https://github.com/AtropsCooper/AttractingContributors-Random

**Comment by aru1702** [comment]
My favorite soccer team is Brazil and Spain, but I prefer Spain.
See my repo >> https://github.com/aru1702/AttractingContributors-TouhouFumo <<

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter
